include/psen_scan/controller_state_machine.h

```diff
@@ -17,45 +17,41 @@
 #define PSEN_SCAN_CONTROLLER_STATE_MACHINE_H
 
 #include <mutex>
+#include <string>
 #include <psen_scan/controller_msm_state_machine.h>
 
 namespace psen_scan
 {
+enum class ProtocolState
```

Comment:
Is this really needed? Is it only for testing? Because

> If what you need simply is doing something with the active state, internal transitions or visitors are a better alternative.

Reply:
Also noticed. Removed.